docs/Guide/CLI/getting-started.md

```diff
@@ -0,0 +1,272 @@
+✔ What's the name of your project? … my-sapphire-bot
+? Choose a language for your project › - Use arrow-keys. Return to submit.
+❯   TypeScript (Recommended)
+    JavaScript
+```
+
+Choose a language for your project, we will choose **TypeScript** in this guide.
+
+```bash {4-6}
+$ sapphire new
+✔ What's the name of your project? … my-sapphire-bot
+✔ Choose a language for your project › TypeScript (Recommended)
+? Choose a template for your project › - Use arrow-keys. Return to submit.
+❯   Default template (Recommended)
+    with Docker
+```
+
+Choose a template for your project, we will go with the **default template** in this guide.
+
+```bash {5-7}
+$ sapphire new
+✔ What's the name of your project? … my-sapphire-bot
+✔ Choose a language for your project › TypeScript (Recommended)
+✔ Choose a template for your project › Default template (Recommended)
+? What format do you want your config file to be in? › - Use arrow-keys. Return to submit.
+    JSON
+❯   YAML
+```
+
+Choose a format for your config file, we will choose **YAML** in this guide. This config file is for Sapphire CLI, read
+more about it here.
+
+```bash {6-8}
+$ sapphire new
+✔ What's the name of your project? … my-sapphire-bot
+✔ Choose a language for your project › TypeScript (Recommended)
+✔ Choose a template for your project › Default template (Recommended)
+✔ What format do you want your config file to be in? › YAML
+? What package manager do you want to use? › - Use arrow-keys. Return to submit.
+❯   Yarn (Recommended)
+    npm
+```
+
+Choose a package manager for your project, we will choose **Yarn** in this guide.
+
+```bash {7}
+$ sapphire new
+✔ What's the name of your project? … my-sapphire-bot
+✔ Choose a language for your project › TypeScript (Recommended)
+✔ Choose a template for your project › Default template (Recommended)
+✔ What format do you want your config file to be in? › YAML
+✔ What package manager do you want to use? › Yarn (Recommended)
+? Do you want to create a git repository for this project? › (y/N)
+```
+
+Lastly, if you want to initialize a git repository in your project, type `y` and enter. If you don't want a git
+repository in your project, just enter.
+
+Now wait for the CLI to create your project and install the dependencies. This should take ~2 minutes.
+
+```bash
+$ sapphire new
+✔ What's the name of your project? … my-sapphire-bot
+✔ Choose a language for your project › TypeScript (Recommended)
+✔ Choose a template for your project › Default template (Recommended)
+✔ What format do you want your config file to be in? › YAML
+✔ What package manager do you want to use? › Yarn (Recommended)
+✔ Do you want to create a git repository for this project? … no
+✔ Cloning the repository
+✔ Setting up the project
+✔ Installing dependencies using Yarn
+Done!
+```
+
+Congratulations, you created a bot using Sapphire CLI. Read the README to see what you need to do before running it.
+
+<Tabs>
+<TabItem value="bash" label="Bash" default>
```

Comment:
This is by no means bash exclusive - preferably refer to it as UNIX shell, or just shell, or anything of that nature so long it isn't a specific exclusive shell.

Reply:
@Favna does docusaurus support posix as a code block language?